Neos.Neos/Resources/Private/Fusion/Prototypes/Shortcut.fusion

```diff
@@ -1,15 +1,133 @@
 # Neos.Neos.Shortcut is given a representation for editing purposes
 #
-prototype(Neos.Neos:Shortcut) < prototype(Neos.Fusion:Template) {
-  templatePath = 'resource://Neos.Neos/Private/Templates/FusionObjects/Shortcut.html'
-
+prototype(Neos.Neos:Shortcut) < prototype(Neos.Fusion:Component) {
   targetMode = ${q(node).property('targetMode')}
   firstChildNode = ${q(node).children('[instanceof Neos.Neos:Document]').get(0)}
   target = ${q(node).property('target')}
   targetConverted = ${Neos.Link.hasSupportedScheme(this.target) ? Neos.Link.convertUriToObject(this.target, node) : null}
   targetSchema = ${Neos.Link.getScheme(this.target)}
 
-  node = ${node}
+  backendLanguage = ${q(Neos.BackendUser.get()).property('preferences.interfaceLanguage')}
+  i18n = ${I18n.id('').package('Neos.Neos').source('Main').locale(this.backendLanguage)}
+
+  renderer = Neos.Fusion:Match {
+    @subject = ${props.targetMode}
+  }
+
+  // targetMode == 'selectedTarget'
```

Comment:
This fusion is pretty hard to read. I would generally recommend:

- merge and nest the conditions
- remove the fallbacks and add them to the english language file so that they are translateable

Reply:
okay thanks ^^
about the language files - the english language file contains most of the sources except for 'shortcut.toSpecificTarget' im sure youre editing the files not per hand - how would i add 'shortcut.toSpecificTarget' ?